src/nock/recorder.test.ts

```diff
@@ -1,81 +1,644 @@
-import { ReplyBody } from 'nock/types';
+import { ApiKeyPlacement, SecurityType } from '@superfaceai/ast';
+import {
+  BoundProfileProvider,
+  ok,
+  ServiceSelector,
+} from '@superfaceai/one-sdk';
+import { define, disableNetConnect, recorder } from 'nock';
+import { mocked } from 'ts-jest/utils';
 
-import { UnexpectedError } from '../common/errors';
+import { BaseURLNotFoundError } from '../common/errors';
+import { writeRecordings } from '../common/output-stream';
+import { mockBoundProfileProvider } from '../superface/mock/boundProfileProvider';
+import { RecordingDefinitions } from '../superface-test.interfaces';
+import * as utils from '../superface-test.utils';
 import {
-  decodeResponse,
-  getRequestHeaderValue,
-  getResponseHeaderValue,
-  parseBody,
-} from './recorder';
-
-describe('Matcher utils', () => {
-  describe('getRequestHeaderValue', () => {
-    it('returns string request header value', () => {
-      const headers = { 'Content-Type': 'application/json' };
-      const expectedResult = 'application/json';
-
-      expect(getRequestHeaderValue('content-type', headers)).toBe(
-        expectedResult
-      );
+  HIDDEN_CREDENTIALS_PLACEHOLDER,
+  HIDDEN_PARAMETERS_PLACEHOLDER,
+} from '../superface-test.utils';
+import { MatchImpact } from './analyzer';
+import { matchTraffic } from './matcher';
+import { endRecording, loadRecording, startRecording } from './recorder';
+import { getRecordings, handleRecordings } from './recorder.utils';
+import { RecordingType, TestRecordings } from './recording.interfaces';
+
+jest.mock('nock');
+
+jest.mock('./recorder.utils', () => ({
+  ...jest.requireActual('./recorder.utils'),
```

Comment:
nitpick: I think you don't need `requireActual` actual as you are importing only Fns that are mocked. Same for other locations.